### PR TITLE
Plans: display regular size domain to plan nudge

### PR DESCRIPTION
**Pull request by gwwar**
As a follow up to #8635 and p3fqKv-40j-p2 this PR renders a regular size nudge on the domains page when:
* A site has a paid domain
* The site has a free plan
* abtest `domainToPersonalPlanNudge3` is set to `nudge`
screen shot 2016-11-09 at 3 11 07 pm

We'll run this test for 2 weeks, using the same baseline as the last test p3fqKv-3LH-p2.

### Testing Instructions:
- Navigate to http://calypso.localhost:3000/domains
- Set the abtest `domainToPersonalPlanNudge3` to `nudge`
- Select a wpcom site that has a paid domain, but a free plan
- We should see a nudge render below domains
- An impression analytics event should fire:
screen shot 2016-11-09 at 11 54 24 am
- Clicking on the nudge takes us to checkout with a personal plan in cart
- The following analytics event should fire:
screen shot 2016-11-09 at 11 55 11 am

cc @rralian @apeatling @lamosty @retrofox

### CODE REVIEW

File: client/blocks/domain-to-plan-nudge/index.jsx [hunk 1]
```diff
@@ -80,9 +79,22 @@ class DomainToPlanNudge extends Component {
...
 
 		page( `/checkout/${ siteId }/personal` );
+	};
+
+	renderRegular() {
+		const { siteId, translate } = this.props;
+		return (
+			<UpgradeNudge
+				title={ translate( 'Upgrade to a Personal Plan and Save!' ) }
+				message={ translate( 'Get live chat support and no ads bundled with your domain!' ) }
```
Comment (lamosty) [inline client/blocks/domain-to-plan-nudge/index.jsx]: "and no ads bundled": I think users could miss the "no" part here and also might not know what an "ad" is without seeing the "WP.com" before it. I would reword it into something like: "and remove WordPress.com Ads".

**Comment by lamosty** [comment]
Working as expected:

selection_200

Apart from that one thing I pointed out (which is not a blocker but a nice thing to have), this LGTM :ship:

**Comment by gwwar** [comment]
Thanks @lamosty! Maybe @meremagee has some ideas for copy?

File: client/blocks/domain-to-plan-nudge/index.jsx (hunk 1 above)
Comment (gwwar) [inline client/blocks/domain-to-plan-nudge/index.jsx]: I'm mostly trying to keep the text to a single line, let's see if folks have some suggestions.

Comment (lamosty) [inline client/blocks/domain-to-plan-nudge/index.jsx]: From what I've seen (and also your screenshot), the text has a long way before jumping to the next line. :) It can survive a few more words IMO.

Comment (gwwar) [inline client/blocks/domain-to-plan-nudge/index.jsx]: It currently barely fits in a mobile view. 😄 Alternatively I could try to fix the nudge to support 2 lines without distorting the svg logo.

Comment (lamosty) [inline client/blocks/domain-to-plan-nudge/index.jsx]: Oh, I completely forgot about mobile. Okay, let's run with this copy then. :)

Comment (gwwar) [inline client/blocks/domain-to-plan-nudge/index.jsx]: Adding some copy help from Krista Stevens:
```
Buy our Personal Plan and remove WordPress.com Ads from your site.
```
